<<<<<<< HEAD
<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NoticeWindow</class>
 <widget class="QWidget" name="NoticeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="noticeButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>通知</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="praiseButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>赞</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="focusButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关注</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="messageButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>80</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>私信</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>701</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255);
font: 22pt &quot;微软雅黑&quot;;
background-color: rgb(255, 185,0);
border-radius:15px;
</string>
   </property>
   <property name="text">
    <string>       消息中心</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
=======
<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NoticeWindow</class>
 <widget class="QWidget" name="NoticeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="noticeButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>通知</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="praiseButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>赞</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="focusButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关注</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="messageButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>80</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>私信</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>701</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255,255,255);
font: 22pt &quot;微软雅黑&quot;;
background-color: rgb(255, 185,0);
border-radius:15px;
</string>
   </property>
   <property name="text">
    <string>       消息中心</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
>>>>>>> cfdf638c3bdfa8efdbbb0c911ad54fe243c63989
